index.html

```diff
@@ -59,10 +59,10 @@
 
   <section id="splash">
     <div class="contentWrapper">
-      <div id="phone">
+      <div id="phone" aria-label="mobile-phone">
```

Comment:
I'm not entirely sure that this label is necessary as it's not an element crucial for accessibility. Can you explain adding it a bit further?

Reply:
Hello Jacob, 

Thank you for your review.

I thought that it may help give more context to the message that was in the element (the one that says 'The Inventor of the 401(k) thinks it has gone awry') layered on top of it. However, it could also be a distraction so I'll remove the label.